moped-editor/src/queries/project.js

```diff
@@ -401,6 +406,12 @@ export const UPDATE_PROJECT_PHASE = gql`
     $object: moped_proj_phases_set_input!
```

Comment:
sidenote but i cant find `UPDATE_PROJECT_PHASE` to be used anywhere in our codebase, do we still need to keep it around?

Reply:
i'm cool with removing since it probably took you time to figure out if it was used or not. It would be nice to reduce confusion, and I don't see our phase mutation needs changing soon. thanks for catching!

Reply:
I agree with removing it if we arent using it anywhere